dGame/dComponents/PetComponent.cpp

```diff
@@ -439,9 +441,15 @@ void PetComponent::Update(float deltaTime) {
 		}
 	}
 
+	auto* missionComponent = owner->GetComponent<MissionComponent>();
+	if (missionComponent == nullptr) return;
+
+	// Determine if "Lost Tags" has been completed and digging has been unlocked
+	const bool digUnlocked = (missionComponent->GetMissionState(842) == eMissionState::COMPLETE);
```

Comment:
Extra parenthesis around the condition, I would also define this 842 to be Lost_Tags in dCommonVars.h since it may be used as a check in other places

Reply:
This would also allow you to remove the comment since you are basically moving that into a constant :)

Reply:
Wouldn't that paste it into every bit of code that includes that header file? I'm not sure that value is really needed outside this one check.

Reply:
hm yeah i dont see it used outside of this file, ill leave it up to you if you want to define the number in an anonymous enum, const variable or leave it alone.

Reply:
I'll also note, dCommonVars.h is pre-compiled, its already included in every translation unit anyways